### ISSUE TITLE
Feature Request: sshkey opstion require password

### ISSUE THREAD

**Issue by oxaro-sean** [nickthecook/crops#12]
Allow configuration options for sshkey dependency to Error and not create key if SSH_KEY_PASSPHRASE is empty (or below a minimum length?)

**Comment by nickthecook** [comment]
Good idea. I'd like to have `ops` error out when there's no passphrase set unless you set an option to allow plaintext private keys.

Need to think a bit about compatibility... technically this may break some current usage of `ops`.
